### ISSUE TITLE
Partition Backfill UI: Unclear what "Re-execute From Last Run" means

### ISSUE THREAD

**Issue by schrockn** [dagster-io/dagster#2989]
I *believe* this has a complicated semantic in that it will retry all partitions where the last run is in a failed state and do a retry-from-failure on that. However this isn't really intuitive. A short name for that is hard to at a minimum we need a tooltip.

<img width="617" alt="Screen Shot 2020-10-02 at 7 03 41 AM" src="https://user-images.githubusercontent.com/28738937/94932088-71b45980-047d-11eb-8256-a90a7c75d59d.png">

**Comment by bengotow** [comment]
Hey! I talked with @sryza about this a week or two ago. I think we should have a few more conversations with customers re: backfills and see if this is at all useful. It sounds like it may be practical for debugging an individual run / solid but too complex to reason about for backfills, even if it was named more clearly...

**Comment by sryza** [comment]
I second what Ben said.  To expand, last time I looked, there were two checkboxes the user can click:

**"Re-execute From Failure"**
This does what Nick described above.  A tooltip we could use for it: "For each partition, if the most recent run failed, launch a re-execution starting from the steps that failed."

I can imagine this being useful in the case where a user encounters an error during a backfill, fixes it, and wants to proceed.

**"Re-execute From Last Run"**
For each partition, if there is a prior run, launches a re-execution.  Otherwise, for that partition, launches an execution.  Executes the subset of steps that are provided by the user.

I believe that this option was more important in the world where we did not support fixed-address intermediates.  We built it as a way to enable users to execute step subsets in a backfill.

IMO, we should take this out for now.  It's possible to imagine situations in which it would be useful, but I'm not confident that they're common enough to merit the pixels / user confusion.  If we decide to keep it, we could use something like the text above as the tooltip.
